common/scala/src/whisk/common/Logging.scala

```diff
@@ -37,24 +39,31 @@ object Verbosity extends Enumeration {
 /**
  * A transaction id.
  */
-case class TransactionId private (id: Long) extends AnyVal {
-    override def toString = if (id == -1) "??" else s"#tid_$id"
+case class TransactionId private (meta: TransactionMetadata) extends AnyVal {
+    override def toString = if (meta.id == -1) "??" else s"#tid_${meta.id}"
 }
 
+case class TransactionMetadata(val id: Long, val first: Instant)
```

Comment:
`first` is the start of the transaction - should be called  `start` or `startTime`? Add javadoc?